### PR TITLE
Added Confirmation Message to Stop-AzDataBoxJob and Remove-AzDataBoxJob

### PR DESCRIPTION
**Pull request by omprakaashve**
## Description
- Added confirmation message to Stop-AzDataBoxJob and Remove-AzDataBoxJob. 
- Added Force switch parameter to skip confirmation in Stop-AzDataBoxJob and Remove-AzDataBoxJob. 
- Made cosmetic changes to help files. 

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/DataBox/DataBox/Common/Cmdlets/StopDataBoxJobCmdlet.cs [hunk 1]
```diff
@@ -48,6 +54,9 @@ public class StopDataBoxJob : AzureDataBoxCmdletBase
         [Parameter(Mandatory = false)]
         public SwitchParameter PassThru;
 
+        [Parameter(Mandatory = false)]
+        public SwitchParameter Force;
```
Comment (markcowl) [inline src/DataBox/DataBox/Common/Cmdlets/StopDataBoxJobCmdlet.cs]: This should not have a Force parameter

File: src/DataBox/DataBox/ChangeLog.md [hunk 2]
```diff
@@ -19,17 +19,10 @@
...
-* Added cmdlet `Get-AzDataBoxCredential`
-	- This cmdlet fetches the databox credentials of the specified databox job.
-* Added cmdlet `Stop-AzDataBoxJob`
-	- This cmdlet is used to cancel an ongoing databox job.
-* Added cmdlet `Remove-AzDataBoxJob`
-	- This cmdlet is used to delete a finished(completed/cancelled/aborted) databox job.
-* Added cmdlet `New-AzDataBoxJob`
-	- This cmdlet is used to create a new databox job.
-
+*Overview of change #1
```
Comment (markcowl) [inline src/DataBox/DataBox/ChangeLog.md]: Why are you removing this?

File: src/DataBox/DataBox.Test/ScenarioTests/DataBoxTests.ps1 [hunk 3]
```diff
@@ -192,8 +191,8 @@ function Test-RemoveDataBoxJob
     try
     {
         Create-Job $dfname $rgname $storageaccount.Id
-		Stop-AzDataBoxJob -ResourceGroupName $rgname -Name $dfname -Reason "Random"
-		Remove-AzDataBoxJob -ResourceGroupName $rgname -Name $dfname 
+		Stop-AzDataBoxJob -ResourceGroupName $rgname -Name $dfname -Reason "Random" -Force
```
Comment (markcowl) [inline src/DataBox/DataBox.Test/ScenarioTests/DataBoxTests.ps1]: Stop should not have Force

File: src/DataBox/DataBox/help/Stop-AzDataBoxJob.md [hunk 4]
```diff
@@ -70,6 +79,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Force
+Force stop without confirmation
```
Comment (markcowl) [inline src/DataBox/DataBox/help/Stop-AzDataBoxJob.md]: Stop should not have Force

File: src/DataBox/DataBox.Test/DataBox.Test.csproj [hunk 5]
```diff
@@ -14,4 +14,8 @@
     <PackageReference Include="Microsoft.Azure.Management.DataBox" Version="1.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (markcowl) [inline src/DataBox/DataBox.Test/DataBox.Test.csproj]: This is already done here: https://github.com/Azure/azure-powershell/blob/master/src/Az.Test.props#L38

File: src/DataBox/DataBox/Common/Cmdlets/StopDataBoxJobCmdlet.cs (hunk 1 above)
Comment (omprakaashve) [reply src/DataBox/DataBox/Common/Cmdlets/StopDataBoxJobCmdlet.cs]: Other Stop cmdlets from other modules also have a Force option. Ex: Stop-AzureVM (https://docs.microsoft.com/en-us/powershell/module/servicemanagement/azure/stop-azurevm?view=azuresmps-4.0.0) . Stop-AzDataBoxJob is used to cancel a job so it needs a optional Force option.

File: src/DataBox/DataBox.Test/DataBox.Test.csproj (hunk 5 above)
Comment (omprakaashve) [reply src/DataBox/DataBox.Test/DataBox.Test.csproj]: Unable to understand what is already done here. Can you please clarify in detail?

Comment (markcowl) [reply src/DataBox/DataBox.Test/DataBox.Test.csproj]: Please look at the file I linked - you are importing it (as all test projects do) and it already copies all of your sessionrecords.  This added itemgroup in your csproj file is redundant.  Remove it, please.

File: src/DataBox/DataBox/ChangeLog.md (hunk 2 above)
Comment (markcowl) [reply src/DataBox/DataBox/ChangeLog.md]: This is a log of all released changes, you should not remove existing changes when adding entries to this file.

File: src/DataBox/DataBox.Test/DataBox.Test.csproj (hunk 5 above)
Comment (omprakaashve) [reply src/DataBox/DataBox.Test/DataBox.Test.csproj]: Thanks for clarification. Removed the redundant entry in csproj file.

File: src/DataBox/DataBox/ChangeLog.md (hunk 2 above)
Comment (omprakaashve) [reply src/DataBox/DataBox/ChangeLog.md]: Got it. Fixed it now by restoring all previous entries.

**Pull request merged**